### ISSUE TITLE
Please add a music switch for new projectors that support this feature

### ISSUE THREAD

**Issue by maulai** [SeydX/homebridge-star-projector#8]
I don't know if this is hard to implement, but if you got the time for it, it would be much appreciated.
Here some examples of projectors with this new feature:
https://www.open-mind-shop.com/products/projektor-3000-version-3-0?variant=42207007244525
https://de.aliexpress.com/item/1005003407752951.html?spm=a2g0o.productlist.0.0.5fcec234mi43TL&algo_pvid=715380e7-04bf-47bb-b0ce-dc8651554e38&algo_exp_id=715380e7-04bf-47bb-b0ce-dc8651554e38-37&pdp_ext_f=%7B"sku_id"%3A"12000025652791377"%7D&pdp_pi=-1%3B46.86%3B-1%3B-1%40salePrice%3BEUR%3Bsearch-mainSearch

**Comment by Fofer** [comment]
Yep, just got this one... which works with this great plug-in... and yes, it supports syncing to music.
https://amzn.to/3JljxVO

I am guessing this is the info @seydx would need to enable a switch for it?
![CleanShot 2022-02-10 at 20 23 59](https://user-images.githubusercontent.com/518749/153537393-ebed4b89-d95d-4e6a-9b85-d91b6f3ed7dd.png)
